src/definitions/elements/emoji.less

```diff
@@ -0,0 +1,67 @@
+/*!
+ * # Fomantic UI - Emoji
+ * http://github.com/fomantic/fomantic-ui/
+ *
+ *
```

Comment:
Remove one line so there is only 2 blank lines

Reply:
:confused: If i remove 1 line, then its 1 line left instead of 2 (because there are only 2? Anyhow, this header is a copy from icon.less, that`s why ;)